packages/block-library/src/social-link/icons/discord.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+export const DiscordIcon = () => (
+	<SVG width="24" height="24" viewBox="0 0 24 24">
```

Comment:
```suggestion
	<SVG width="24" height="24" viewBox="0 0 24 24" version="1.1">
```

Nit: There's no visual difference, but it would be nice to match the format with the other icons.